app/views/users/_admin_change_username.html.erb

```diff
@@ -0,0 +1,27 @@
+<h2 class="heading"><%= t(".page_heading") %></h2>
+<%= error_messages_for :user %>
+<p class="note"><%= t(".description") %></p>
+
+<ul class="navigation actions">
+  <li><%= link_to t(".navigation.manage_user"), admin_user_path(@user) %></li>
+</ul>
+
+<%= form_tag changed_username_user_path(@user), autocomplete: "off" do %>
+  <dl>
```

Comment:
Damn, I just realized we're missing legends and headings on this and the original form. We should fix that:

```suggestion
<%= form_tag changed_username_user_path(@user), class: "simple post", autocomplete: "off" do %>
  <legend><%= t("users.change_username.legend") %></legend>
  <h3 class="landmark heading"><%= t("users.change_username.heading") %></h3>
  <dl>
```

For the actual text , "Change Username" is fine.